### PR TITLE
V1.read data

### PR DESCRIPTION
**Pull request by ktrinanm**
Hey guys. If you don't mind checking stuff like logic or missing stuff or anything else.

### CODE REVIEW

File: ClassificationNeuralNetwork.java [hunk 1]
```diff
@@ -1,8 +1,65 @@
...
+		concavity = new double[numLines];
+		concavePnts = new double[numLines];
+		symmetry = new double[numLines];
+		fractalDim = new double[numLines];
+		int currLineNum = 0;
+
+		try
+		{
+			FileReader fileReader = new FileReader(dataFileName);
+			BufferedReader bufferedReader
```
Comment (ghost) [inline ClassificationNeuralNetwork.java]: Just one suggestion: you could probably combine these two lines together as the following:

BufferedReader bufferedReader= new BufferedReader(new FileReader(dataFileName));

Comment (ghost) [reply ClassificationNeuralNetwork.java]: Aside from that, it looks good.

**Comment by justintimefortech** [comment]
Also, upon looking at the code again, why are the doubles taken as parameters to readDataFromFile?

**Comment by ktrinanm** [comment]
Because I wasn't going to make them member variables. I'm going to do some
testing tonight and I might change my mind..

**Comment by justintimefortech** [comment]
Alright, that looks good. I'll get onto the shuffle method.

**Pull request merged**